definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -6,10 +6,12 @@
 - Electricity Storage Type:
 
   - Energy Storage System:
-      description: Mainly to refer to the batteries
+      description: all types of energy storage including pumped hydro and batteries
+  - Energy Storage System|Pumped Hydro Storage:
```

Comment:
Why not call this Energy Storage System|Hydro|Pumped Storage? Would be more consistent with the way hydro is defined

Reply:
Updated as suggested for consistency